experimental/framework/data-objects/api-extractor.json

```diff
@@ -1,4 +1,13 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-	"extends": "@fluidframework/build-common/api-extractor-base.json"
+	"extends": "../../../common/build/build-common/api-extractor-base.json",
+	"messages": {
+		"extractorMessageReporting": {
+			// TODO: Add missing documentation and remove this rule override
+			"ae-undocumented": {
+				"logLevel": "error",
+				"addToApiReportFile": true
+			}
```

Comment:
No md file with new warnings? Is the override required here?

Reply:
I'll double check these. It's possible I added to the wrong packages.

Reply:
It turns out this config wasn't used - this package doesn't currently use API-Extractor. I'm betting one of @jikim-msft's PRs will likely add it soon. For now, I'm just going to delete this config. We can re-add one as needed.